pkg/assisted/list.go

```diff
@@ -0,0 +1,67 @@
+package assisted
+
+import (
+	"context"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	assistedv1beta1 "github.com/openshift/assisted-service/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// ListNmStateConfigsInAllNamespaces returns a a cluster-wide NMStateConfig list.
+func ListNmStateConfigsInAllNamespaces(apiClient *clients.Settings) ([]*NmStateConfigBuilder, error) {
+	nmStateConfigList := &assistedv1beta1.NMStateConfigList{}
+
+	err := apiClient.List(context.Background(), nmStateConfigList, &client.ListOptions{})
+
+	if err != nil {
+		glog.V(100).Infof("Failed to list nmStateConfigs due to %s", err.Error())
```

Comment:
Please update to something like:
glog.V(100).Infof("Failed to list nmStateConfigs across all namespaces due to %s", err.Error())